src/sentry/options/defaults.py

```diff
@@ -569,6 +569,10 @@
 register("performance.issues.n_plus_one_api_calls.la-rollout", default=0.0)
 register("performance.issues.n_plus_one_api_calls.ea-rollout", default=0.0)
 register("performance.issues.n_plus_one_api_calls.ga-rollout", default=0.0)
+register("performance.issues.slow_span.problem-creation", default=0.0)
+register("performance.issues.slow_span.la-rollout", default=0.0)
+register("performance.issues.slow_span.ea-rollout", default=0.0)
+register("performance.issues.slow_span.ga-rollout", default=0.0)
```

Comment:
You don't need to declare any of these options! You can override `SlowSpanDetector.is_creation_allowed_for_system` and have it always return `True`